docs/security/access-control.md

```diff
@@ -0,0 +1,659 @@
+|--------------|----------------------------------|-------------------------------------------|
+| CreateTopic  | Metalake, Catalog, Schema        | Create a topic                            |
+| ProduceTopic | Metalake, Catalog, Schema, Topic | Produce a topic (including alter a topic) |
+| ConsumeTopic | Metalake, Catalog, Schema, Topic | Consume a topic                           |
+
+### Fileset privileges
+
+| Name          | Supports Securable Object          | Operation                                   |
+|---------------|------------------------------------|---------------------------------------------|
+| CreateFileset | Metalake, Catalog, Schema          | Create a fileset                            |
+| WriteFileset  | Metalake, Catalog, Schema, Fileset | Write a fileset (including alter a fileset) |
+| ReadFileset   | Metalake, Catalog, Schema, Fileset | read a fileset                              |
+
+## Inheritance Model
+
+Securable objects in Gravitino are hierarchical and privileges are inherited downward.
+
+This means that granting a privilege on a metalake, catalog or schema automatically grants
+the privilege to all current and future objects within the metalake, catalog or schema.
+
+For example, if you give a use that `SELECT_TABLE` privilege on a catalog, then that the user
+will be able to select(read) all tables in that catalog.
+
+## Privilege Condition
+
+The privilege supports two condition: `allow` and `deny`. `allow` means that you are able to use the privilege,
+
+`deny` means that you aren't able to use the privilege.
+
+`deny` condition is prior to `allow` condition. If a role has the `allow` condition and `deny` condition at the same time. 
+The user won't be able to use the privilege.
+
+If parent securable object has the same privilege name with different condition, the parent securable privilege will still take effect.
+
+For example, securable metalake object allows to use the catalog, but securable catalog denies to use the catalog, the user isn't able to use the catalog.
+
+If securable metalake object denies to use the catalog, but securable catalog allows to use the catalog, the user isn't able to use the catalog, too.
+
+![privileg_image](../assets/privilege.png)
+
+## Server Configuration
+
+If you want to enable the access control, you should enable the authorization.
+
+The related configuration is as follows.
+
+| Configuration item                       | Description                                           | Default value | Required                         | Since Version |
+|------------------------------------------|-------------------------------------------------------|---------------|----------------------------------|---------------|
+| `gravitino.authorization.enable`         | Enable the authorization                              | false         | No                               | 0.5.0         |
```

Comment:
`Enable the authorization` -> Whether Gravtino enable authorization or not

Reply:
Done.